src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -3,33 +3,74 @@ import React, { FC } from "react"
 import styled from "styled-components"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Details_conversation } from "v2/__generated__/Details_conversation.graphql"
+import { Media } from "v2/Utils/Responsive"
 
-const BorderedFlex = styled(Flex)`
+export const DETAIL_BOX_ANIMATION = `transition: width 0.3s ;`
+const DETAIL_BOX_XS_ANIMATION = `transition: opacity 0.3s;`
+
+const DetailsContainer = styled(Flex)<{ opacity?: 0 | 1 }>`
   border-left: 1px solid ${color("black10")};
   flex-shrink: 0;
-  margin-left: -1px;
+  ${DETAIL_BOX_ANIMATION}
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  background-color: ${color("white100")};
+`
+// XS screen has opacity animation instead of width
+const XSDetailsContainer = styled(DetailsContainer)`
+  ${DETAIL_BOX_XS_ANIMATION}
+  opacity: ${({ opacity }) => opacity};
 `
-
 interface DetailsProps extends FlexProps {
   conversation: Details_conversation
+  showDetails: boolean
 }
 
 export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
-  return (
-    <BorderedFlex
-      flexDirection="column"
-      justifyContent="flex-start"
-      flexShrink={0}
-      {...props}
-    >
+  const content = (
+    <>
       <EntityHeader
         px={2}
         py={1}
         name={conversation.to.name}
         initials={conversation.to.initials}
       />
       <Separator />
-    </BorderedFlex>
+    </>
+  )
+
+  return (
+    <>
```

Comment:
For screen sizes larger than `xs` width animates from `0 `to `375px`. For `xl` screens position is `static` so it pushes the middle column content left but for other screen sizes (x/m/l) it opens over the middle column

Reply:
I'm not suggesting you make a change here, but typically you want to avoid animating things like width & height. Animating the dimensions of an element causes continual repaint/reflows as the element is animating which can cause the animation to be choppy. A better alternative (in most cases) is to use `transform: translate(-375px)` (or whatever the equivalent for the situation). Translate moves where the container renders by utilizing GPU manipulations which can be substantially faster and smoother (especially on mobile which isn't necessarily a concern for this particular animation).

Reply:
Second this and would encourage we use transform versus pushing this off. Depending on the users browser we could get drastic differences in performance. By using translate3d we instantly tap into the GPU in a cross-browser / cross-platform kind of way.

Reply:
hmmm, the problem is the `xl` screens. `translate` doesn't seem to push the content of the middle column.

Reply:
I can change it to use `translate` for `s/md/lg` screens but keep using `width` animation for `xl` screens (which are most likely more powerful computers). It makes the code more complicated but more performant on `s/md/lg`. Does it worth it?

Reply:
Yeah i'm mostly thinking bad android tablets. I haven't done any substantial web animation since 2013 but back then animating with transform was necessary. Maybe its not so much anymore for something simple like this as so much has changed. But whatever you think is best! don't want to hold this up.

Reply:
Thanks for the suggestions. in  https://github.com/artsy/force/pull/5735/commits/9e6f796b1f271f4b4fd5ca4d24844ed4301429a0